PowerShellGet/PSModule.psm1

```diff
@@ -4906,6 +4906,16 @@ function Test-ScriptFileInfo
             return
         }
 
+        if ($PSScriptInfo.Version -match '-')

```

Comment:
I am ok with that.  The prerelease string is validated in more detail (including checking for more than one hyphen) using Validate-PreReleaseString.